macros/src/test/scala/CostFunction.scala

```diff
@@ -82,24 +82,24 @@ circuit target_memory :
     mem_0_0.addr <= addr
     node dout_0_0 = bits(mem_0_0.dout, 31, 0)
     mem_0_0.din <= bits(din, 31, 0)
-    mem_0_0.write_en <= and(and(write_en, UInt<1>("h1")), UInt<1>("h1"))
+    mem_0_0.write_en <= and(and(and(write_en, UInt<1>("h1")), UInt<1>("h1")), UInt<1>("h1"))
     mem_0_1.clk <= clk
     mem_0_1.addr <= addr
     node dout_0_1 = bits(mem_0_1.dout, 31, 0)
     mem_0_1.din <= bits(din, 63, 32)
-    mem_0_1.write_en <= and(and(write_en, UInt<1>("h1")), UInt<1>("h1"))
+    mem_0_1.write_en <= and(and(and(write_en, UInt<1>("h1")), UInt<1>("h1")), UInt<1>("h1"))
     mem_0_2.clk <= clk
     mem_0_2.addr <= addr
     node dout_0_2 = bits(mem_0_2.dout, 31, 0)
     mem_0_2.din <= bits(din, 95, 64)
-    mem_0_2.write_en <= and(and(write_en, UInt<1>("h1")), UInt<1>("h1"))
+    mem_0_2.write_en <= and(and(and(write_en, UInt<1>("h1")), UInt<1>("h1")), UInt<1>("h1"))
     mem_0_3.clk <= clk
     mem_0_3.addr <= addr
     node dout_0_3 = bits(mem_0_3.dout, 31, 0)
     mem_0_3.din <= bits(din, 127, 96)
-    mem_0_3.write_en <= and(and(write_en, UInt<1>("h1")), UInt<1>("h1"))
+    mem_0_3.write_en <= and(and(and(write_en, UInt<1>("h1")), UInt<1>("h1")), UInt<1>("h1"))
```

Comment:
I assume this is due to a FIRRTL change, but where did this extra redundant and with 1 come from?

Reply:
Some of the and's are form #69 and some are from #70.
The width changes are form #74